targets/targets.json

```diff
@@ -3440,6 +3440,29 @@
             }
         }
     },
+    "MTB_LAIRD_BL652": {
+        "inherits": ["MCU_NRF52"],
+        "macros_add": ["BOARD_PCA10040", "NRF52_PAN_12", "NRF52_PAN_15", "NRF52_PAN_58", "NRF52_PAN_55", "NRF52_PAN_54", "NRF52_PAN_31", "NRF52_PAN_30", "NRF52_PAN_51", "NRF52_PAN_36", "NRF52_PAN_53", "S132", "CONFIG_GPIO_AS_PINRESET", "BLE_STACK_SUPPORT_REQD", "SWI_DISABLE0", "NRF52_PAN_20", "NRF52_PAN_64", "NRF52_PAN_62", "NRF52_PAN_63"],
+        "device_has_add": ["ANALOGIN", "I2C", "I2C_ASYNCH", "INTERRUPTIN", "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "SLEEP", "SPI", "SPI_ASYNCH", "SPISLAVE"],
+        "extra_labels_add": ["MTB_LAIRD_BL652"],
+        "release_versions": ["5"],
+        "device_name": "nRF52832_xxAA",
+        "bootloader_supported": true,
+        "config": {
+            "usb_tx": {
+                "help": "Value SIO_06",
+                "value": "SIO_6"
+            },
+            "usb_rx": {
+                "help": "Value SIO_08",
+                "value": "SIO_8"
+            }
```

Comment:
What are usb_tx and usb_rx being used for?

Reply:
@marcuschangarm ..they are used for the DAPLink .. https://github.com/ARMmbed/mbed-os/pull/6164/files#diff-67ec7d72556342b67f0cb5b4abfe4516R200

Reply:
But none of our code base uses the USB_RX configuration flag. It looks like dead code.

Reply:
Isn't USB_TX and USB_RX standardised pins to be defined as part of DAPLink pins in the pin defines? along with BUTTON and LED's..?

Reply:
STDIO_UART_TX and STDIO_UART_RX are the only pins being used.

https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52832/PeripheralNames.h#L48-L54

https://github.com/ARMmbed/mbed-os/blob/master/platform/mbed_retarget.cpp#L220

Reply:
Ah, I see.. good to know. I'll create a different PR to optimise this target then. Thanks!

Reply:
I wouldn't worry about it to be honest! 😄 We are very close to merging the NRF52 feature branch anyway and we'll be cleaning up all the NRF52 based targets at that point!